.github/workflows/jira-lint-and-link.yml

```diff
@@ -13,4 +13,4 @@ jobs:
           jira-token: ${{ secrets.JIRA_TOKEN }}
           jira-base-url: https://wearezeta.atlassian.net
           skip-branches: '^(production-release|main|master|release\/v\d+)$' #optional
-          fail-when-jira-issue-not-found: true
+          fail-when-jira-issue-not-found: false
```

Comment:
You can even remove it since it's the default value (cf https://github.com/cakeinpanic/jira-description-action#options)